### PR TITLE
Create a dummy version of a users (no password or authentication)

### PR DESCRIPTION
**Pull request by theowiik**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+  n.times do
     SlotMachineSymbol.create(
       name: Faker::SlackEmoji.emoji,
       slot_machine: slot_machine
     )
   end
 end
+
+puts('Created a sample user...')
+User.create(name: 'Joe Danger', email: 'joe@danger.com')
```
Comment (Victor-Ek) [inline db/seeds.rb:21]: 😍

File: Gemfile [hunk 2]
```diff
@@ -8,6 +8,8 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.5'
 gem 'dotenv', '~> 2.1', '>= 2.1.1'
+gem 'knock', '~> 2.1', '>= 2.1.1'
```
Comment (Victor-Ek) [inline Gemfile:11]: !!!
GET THIS OFF MY GEMFILE!

Comment (theowiik) [reply Gemfile:11]: Sir, yes sir ✋😨

Comment (theowiik) [reply Gemfile]: Sir, fixed in this commit 3d6d14e

Comment (Victor-Ek) [reply Gemfile]: Oh I see how nice. I'll see to it that I approve! :)

**Pull request merged**